packages/labs/context/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# @lit-labs/context
+
+## 1.0.0
```

Comment:
Are we intentionally publishing this and going straight to 1.0.0?

Reply:
per offline discussion, we'll start with 0.1.0